recipes/yaml-cpp4rkt/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+MACOSX_SDK_VERSION:        # [osx]
```

Comment:
Why is this needed? This isn't used in the `yaml-cpp` feedstock.

Reply:
Inherited from other recipes in which this was needed. I'll remove this.

Reply:
Removed!